packages/coil-extension/src/content/services/ContentScript.ts

```diff
@@ -76,14 +76,31 @@ export class ContentScript {
       this.runtime.sendMessage(request)
     }
 
+    const associatePaymentPointer = (paymentPointer: string) => {
```

Comment:
We ought to be able to piggy back the existing start/stop commands though?
there shouldn't be need for any new messages